tests/test_ghosting.py

```diff
@@ -79,44 +85,57 @@ def test_get_ghosting(self):
+        range(min(ELIGIBLE_GHOST_AREA[1]), max(ELIGIBLE_GHOST_AREA[1])), dtype=np.intp)[:, np.newaxis], np.array(
+        range(min(ELIGIBLE_GHOST_AREA[0]), max(ELIGIBLE_GHOST_AREA[0]))
+    )
+        PE = "COL"
+        GHOSTING = (None,  0.015138960417776908)
 
-    def setUp(self):
-        self.file = str(TEST_DATA_DIR / 'ghosting' / 'PE_COL_PHANTOM_BOTTOM_RIGHT' / 'PE_COL_PHANTOM_BOTTOM_RIGHT.IMA')
-        self.dcm = pydicom.read_file(self.file)
+        def setUp(self):
+            self.file = str(
+                TEST_DATA_DIR / 'ghosting' / 'PE_COL_PHANTOM_BOTTOM_RIGHT' / 'PE_COL_PHANTOM_BOTTOM_RIGHT.IMA')
+            self.dcm = pydicom.read_file(self.file)
 
 
 class TestAxialPhilipsBroomfields(TestGhosting):
-    SIGNAL_BOUNDING_BOX = (217, 299, 11, 93)
-    SIGNAL_CENTRE = [(SIGNAL_BOUNDING_BOX[0] + SIGNAL_BOUNDING_BOX[1]) // 2,
-                     (SIGNAL_BOUNDING_BOX[2] + SIGNAL_BOUNDING_BOX[3]) // 2]
-    BACKGROUND_ROIS = [(258, 264), (194, 264), (130, 264), (66, 264)]
-    PADDING_FROM_BOX = 30
-    SLICE_RADIUS = 5
-    ELIGIBLE_GHOST_AREA = range(SLICE_RADIUS, SIGNAL_BOUNDING_BOX[0] - PADDING_FROM_BOX), range(
-        SIGNAL_BOUNDING_BOX[2], SIGNAL_BOUNDING_BOX[3])
-    SIGNAL_SLICE = np.array(range(SIGNAL_CENTRE[0] - SLICE_RADIUS, SIGNAL_CENTRE[0] + SLICE_RADIUS), dtype=np.intp)[:,
-                   np.newaxis], np.array(range(SIGNAL_CENTRE[1] - SLICE_RADIUS, SIGNAL_CENTRE[1] + SLICE_RADIUS),
-                                         dtype=np.intp)
-    GHOST_SLICE = np.array(range(11 - SLICE_RADIUS, 11 + SLICE_RADIUS), dtype=np.intp)[:, np.newaxis], np.array(
-        range(16 - SLICE_RADIUS, 16 + SLICE_RADIUS))
+        SIGNAL_BOUNDING_BOX = (217, 299, 11, 93)
+        SIGNAL_CENTRE = [(SIGNAL_BOUNDING_BOX[0] + SIGNAL_BOUNDING_BOX[1]) // 2,
+                         (SIGNAL_BOUNDING_BOX[2] + SIGNAL_BOUNDING_BOX[3]) // 2]
+        BACKGROUND_ROIS = [(258, 264), (194, 264), (130, 264), (66, 264)]
+        PADDING_FROM_BOX = 30
+        SLICE_RADIUS = 5
+        ELIGIBLE_GHOST_AREA = range(SLICE_RADIUS, SIGNAL_BOUNDING_BOX[0] - PADDING_FROM_BOX), range(
+            SIGNAL_BOUNDING_BOX[2], SIGNAL_BOUNDING_BOX[3])
+        SIGNAL_SLICE = np.array(range(SIGNAL_CENTRE[0] - SLICE_RADIUS, SIGNAL_CENTRE[0] + SLICE_RADIUS), dtype=np.intp)[
+                       :,
+                       np.newaxis], np.array(range(SIGNAL_CENTRE[1] - SLICE_RADIUS, SIGNAL_CENTRE[1] + SLICE_RADIUS),
+                                             dtype=np.intp)
+        GHOST_SLICE = np.array(
+        range(min(ELIGIBLE_GHOST_AREA[1]), max(ELIGIBLE_GHOST_AREA[1])), dtype=np.intp)[:, np.newaxis], np.array(
+        range(min(ELIGIBLE_GHOST_AREA[0]), max(ELIGIBLE_GHOST_AREA[0]))
+    )
+
+        GHOSTING = (None,  0.007246960909896829)
+
+        def setUp(self):
+            self.file = str(TEST_DATA_DIR / 'ghosting' / 'GHOSTING' / 'axial_philips_broomfields.dcm')
+            self.dcm = pydicom.read_file(self.file)
+
+
+
+
```

Comment:
A lot of this indentation seems to be double-tabbed. Is that intentional? Github thinks the code has changed, but a lot of it looks the same from what I can tell.

Reply:
I'll tidy up the indentation.